yaksh/static/yaksh/js/requesthandler.js

```diff
@@ -0,0 +1,139 @@
+request_status = "initial"
+count = 0;
+checker = null
+function submitRequest(){
+    document.forms["code"].submit();
+}
+
+function check_state(state, uid) {
+    if ((state == "running" || state == "not started") && count < 7) {
+        count++;
+        setTimeout(function() {get_result(uid);}, 2000);
+    } else if (state == "unknown") {
+        request_status = "initial";
+        count = 0;
+        notify("Request timeout. Try again later");
+        clearInterval(checker);
```

Comment:
Ugly.

Reply:
Removed setInterval, actually had added to debug selenium tests.
Not needed as all the conditions are checked.